app/models/form/sales/pages/housing_benefits.rb

```diff
@@ -2,7 +2,14 @@ class Form::Sales::Pages::HousingBenefits < ::Form::Page
   def initialize(id, hsh, subsection, joint_purchase:)
     super(id, hsh, subsection)
     @joint_purchase = joint_purchase
-    @depends_on = [{ "jointpur" => @joint_purchase ? 1 : 2 }]
+    @depends_on =
```

Comment:
it works but think i would extract to a method at this point. not sure how i feel having an `if` within an array?